### PR TITLE
Remove out-dated pipeline link

### PR DESCRIPTION
**Pull request by liliankasem**
The project is out-dated and will be deleted; I would like any references to my private AzDO to be removed

AB#16103546

### CODE REVIEW

**Comment by liliankasem** [comment]
Hey @hartra344, do you think you can help me merge this PR?

File: azure-devops-sample/README.md [hunk 1]
```diff
@@ -115,8 +115,6 @@ We can now test out our new connection!
 
 ## DevOps
```
Comment (hartra344) [inline azure-devops-sample/README.md:116]: Since there's nothing here now can you go ahead and remove this header entirely?

**Review by hartra344 — APPROVED** [review]

File: azure-devops-sample/README.md (hunk 1 above)
Comment (liliankasem) [reply azure-devops-sample/README.md:116]: I don't think we should; this is a high level heading similar to `## Local` above - all the headings below this fall into the "DevOps" category for deployment, similar to how every heading under "Local" falls into local deployment

Comment (hartra344) [reply azure-devops-sample/README.md:116]: Oops I misread it, you're totally correct. Merging now. Thanks for updating :)

Comment (liliankasem) [reply azure-devops-sample/README.md:116]: Thanks for the fast response :D

**Pull request merged**